document/src/main/java/com/ritense/document/web/rest/impl/SearchFieldResource.java

```diff
@@ -53,4 +54,15 @@ public ResponseEntity<List<SearchField>> getSearchField(
         @PathVariable String documentDefinitionName) {
         return ResponseEntity.ok(searchFieldService.getSearchFields(documentDefinitionName));
     }
+
+    @Override
+    @PutMapping("/v1/document-search/{documentDefinitionName}/fields")
```

Comment:
This endpoint should also be added to the SecurityConfiguration. See:
https://github.com/valtimo-platform/valtimo-backend-libraries/blob/development/document/src/main/java/com/ritense/document/security/config/DocumentSearchHttpSecurityConfigurer.java#L33

Reply:
added